src/content/docs/develop/configuration-files.mdx

```diff
@@ -71,6 +77,103 @@ endpoints = ["https://my.app.updater/{{target}}/{{current_version}}"]
 
 Note that JSON5 and TOML supports comments, and TOML can use kebab-case for config names which are more idiomatic.
 
+### Platform-specific Configuration
+
+In addition to the default configuration file, Tauri can read a platform-specific configuration from:
+
+- `tauri.linux.conf.json` or `Tauri.linux.toml` for Linux
+- `tauri.windows.conf.json` or `Tauri.windows.toml` for Windows
+- `tauri.macos.conf.json` or `Tauri.macos.toml` for macOS
+- `tauri.android.conf.json` or `Tauri.android.toml` for Android
+- `tauri.ios.conf.json` or `Tauri.ios.toml` for iOS
+
+The platform-specific configuration file gets merged with the main configuration object following the [JSON Merge Patch (RFC 7396)] specification.
```

Comment:
```suggestion
The platform-specific configuration file gets merged with the main configuration object following the [JSON Merge Patch (RFC 7396)](https://www.rfc-editor.org/info/rfc7396) specification.
```

Reply:
didn't i include this link at the bottom of the page?

Reply:
my bad, I didn't see it.